protocols/gossipsub/src/protocol.rs

```diff
@@ -61,6 +61,8 @@ pub(crate) const FLOODSUB_PROTOCOL: ProtocolId = ProtocolId {
     kind: PeerKind::Floodsub,
 };
 
+pub const DEFAULT_MAX_TRANSMIT_SIZE: usize = 65536;
```

Comment:
same as above, do we need this to be `pub`?

Reply:
There's a failure otherwise https://github.com/libp2p/rust-libp2p/actions/runs/13655153154/job/38172407946

Reply:
and doesn't `pub(crate)` work?

Reply:
That's what I had when it failed https://github.com/libp2p/rust-libp2p/pull/5906/commits/1e72f140f34da79003d5ba27450268318ace7f99